### PR TITLE
Add medications to reminders

### PR DESCRIPTION
**Pull request by sbotor**

### CODE REVIEW

File: app/src/main/java/com/sbcf/pillbox/features/medicationreminders/components/ReminderMedicationListItem.kt
Comment (MrChrisFix) [inline app/src/main/java/com/sbcf/pillbox/features/medicationreminders/components/ReminderMedicationListItem.kt:1]: When someone gets the notification the list looks a bit blank. I would add the dosage beside the name (if possible), which could be a QoL improvement

File: app/src/main/java/com/sbcf/pillbox/features/medicationreminders/screens/MedicationReminderEditScreen.kt
Comment (MrChrisFix) [inline app/src/main/java/com/sbcf/pillbox/features/medicationreminders/screens/MedicationReminderEditScreen.kt:1]: I am not sure, if the current layout is UX friendly on smaller devices. The medicines list is quite small compared to the date setter

**Review by MrChrisFix — COMMENTED** [review]
Will approve after comments responses

File: app/src/main/java/com/sbcf/pillbox/features/medicationreminders/screens/MedicationReminderEditScreen.kt
Comment (sbotor) [reply app/src/main/java/com/sbcf/pillbox/features/medicationreminders/screens/MedicationReminderEditScreen.kt:1]: yes

Comment (sbotor) [reply app/src/main/java/com/sbcf/pillbox/features/medicationreminders/screens/MedicationReminderEditScreen.kt:1]: and there is no indication that the list is scrollable

**Review by MrChrisFix — APPROVED** [review]

**Review by MrChrisFix — APPROVED** [review]
Approved new changes

**Pull request merged**